Makefile

```diff
@@ -107,17 +107,29 @@ kill:
 
 # http://stackoverflow.com/questions/2214575/passing-arguments-to-make-run#2214593
 ifeq (run,$(firstword $(MAKECMDGOALS)))
-  # use the rest as arguments for "run"
-  RUN_ARGS := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+  # https://stackoverflow.com/questions/10571658/gnu-make-convert-spaces-to-colons
+  space :=
+  space +=
+  # From word 2 to the end is the task
+  TASK := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+  # Split the quelified task into words and remove the last word (task name) to get the module name
+  MOD_NAME := $(subst $(space),.,$(filter-out $(word $(words $(subst ., ,$(wordlist 2,2,$(MAKECMDGOALS)))),$(subst ., ,$(wordlist 2,2,$(MAKECMDGOALS)))), $(subst ., ,$(wordlist 2,2,$(MAKECMDGOALS)))))
```

Comment:
Wow! That Makefile syntax!

Trying to see it from the outside, I asked Google how we could run shell commands in here. You can do something like `MOD_NAME != command`

Maybe something like `echo $(TASK) | sed "s/^\(.*\)\..*$/\1/"`? Still not very readable, but at least is just a regex, not some custom `make` thing.